common/templates/student/edx_ace/emailchangeconfirmation/email/body.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>
+                {% trans "Email Change Confirmation" %}
+            </h1>
+            <p style="color: rgba(0,0,0,.75);">
+                {% blocktrans %}This is to confirm that you changed the e-mail associated with {{ platform_name }} from {{ old_email }} to {{ new_email }}. If you did not make this request, please contact us immediately. Contact information is listed at:{% endblocktrans %}
```

Comment:
Does `platform_name` get substituted?

Reply:
I don't know if a understood your question correctly; I think the below is the answer you are looking for (I hope :P ):

`ace.send(msg)` in `confirm_email_change` function in **common/djangoapps/student/views/management.py** will send the `msg` object to ace to do the magic

`msg` is being created by the ace personalize function
`msg = EmailChangeConfirmation().personalize( ...`
There you will see a `user_context` filled from `message_context` object

Now, `message_context` is created by `get_base_template_context` that fills it with the `platform_name` and many other values